code/modules/mechs/premade/misc.dm

```diff
@@ -25,6 +26,9 @@
 	mech_step_sound = 'sound/mechs/tanktread.ogg'
 	can_strafe = FALSE
 	can_climb = FALSE
+	shielding = 15
+
+	side_mult = 1.2
```

Comment:
Are tank treads more vulnerable from the front? Wouldn't their profile be too small?